scifight/models.py

```diff
@@ -16,6 +20,16 @@
     hold a number plus possible brief one-word explanation. """
 
 
+class Tournament(models.Model):
+    name        = models.CharField(max_length=NAME_LENGTH)
+    slug        = models.SlugField(max_length=SLUG_LENGTH, unique=True)
+    description = models.TextField(blank=True, null=True)
+    create_data = models.DateTimeField(default=timezone.now)
```

Comment:
Maybe have `short_name` field, too?
For example:
* if `name` is «International Physicists' Tournament — 2032»
* then `short_name` is «IPT'2032».
In this case, it's better to rename `name` to `full_name`.